<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explore</class>
 <widget class="QDialog" name="Explore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="trv_dir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lsw_file">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>256</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_drivers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
